packages/product-json-to-csv/test/map-headers.spec.js

```diff
@@ -122,10 +122,10 @@ describe('mapHeaders', () => {
       })
     })
 
-    test('contains `attribute`', () => {
+    test('contains `attributes`', () => {
```

Comment:
Same question here, does it mean that singular `attribute` is not a thing?

Reply:
Same as above.